.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      
+      - name: Setup
+        uses: actions/setup-node@094c36e88e2a3ffdf4d3a5bb935088ac39a46acc
+        with:
+          node-version: 16
+      
+      - name: Install
+        run: npm ci
+
```

Comment:
Can we also do an `npm test`?

Reply:
Sure, but there aren't any tests.

Reply:
It'll remind me to run `prettier` at the very least.